packages/deployed-backend-client/API.md

```diff
@@ -173,22 +187,14 @@ export type FunctionConfiguration = {
 };
 
 // @public (undocumented)
-export type ListSandboxesRequest = {
-    nextToken?: string;
-};
-
-// @public (undocumented)
-export type ListSandboxesResponse = {
-    sandboxes: SandboxMetadata[];
-    nextToken: string | undefined;
+export type ListBackendsRequest = {
+    deploymentType: DeploymentType;
+    backendStatusFilters?: BackendStatus[];
 };
 
 // @public (undocumented)
-export type SandboxMetadata = {
-    name: string;
-    lastUpdated: Date | undefined;
-    status: BackendDeploymentStatus;
-    backendId: BackendIdentifier | undefined;
+export type ListBackendsResponse = {
+    getBackendSummaryByPage: AsyncGenerator<BackendSummaryMetadata[]>;

```

Comment:
If this is just property then it should be named `backendSummaryByPage`.
If we want to keep `getBackendSummaryByPage` then this should become 
`getBackendSummaryByPage: () => AsyncGenerator<BackendSummaryMetadata[]>;`

Fwiw. I think this should be a function.